aws/handlers/appoinments_api_req_handler.go

```diff
@@ -15,23 +13,33 @@ import (
 
 // HandleAppointmentsRequest will handle all appointment gate api requests
 func HandleAppointmentsRequest(req events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
-	db, err := database.CreateDB()
-
-	if err != nil {
-		return utils.GatewayResponse(http.StatusBadRequest, models.ErrorBody{ErrorMsg: aws.String(err.Error())})
-	}
-
-	repository := repositories.AppointmentsRepository{DB: db}
 
-	//TODO
-	counselor := counselor.Counselor{
-		Identifier: "random",
+	switch req.HTTPMethod {
+	case "GET":
+		db, err := database.CreateDB()
+
+		if err != nil {
+			return utils.GatewayResponse(http.StatusBadRequest, models.ErrorBody{ErrorMsg: aws.String(err.Error())})
+		}
+
+		repository := repositories.AppointmentsRepository{DB: db}
+		counselors, err := repository.FetchAll(nil)
```

Comment:
i think getting all appointments may not make sense right... we would want to either get all appointments for a counselor or a user between given range of dates... that's reason for including counselor as an argument back then

Reply:
this is still work in progress , will add a comment